.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
-  "python.formatting.provider": "black",
```

Comment:
Sorry, but we require all contributors to use the same pre-commit rules.

Reply:
just updated, please revise

Reply:
I still see the provider as autopep8, when it should be `black`. do you maybe have a local commit that you didn't push yet?